src/nvim/eval.c

```diff
@@ -351,7 +351,6 @@ typedef enum {
  * The reason to use this table anyway is for very quick access to the
  * variables with the VV_ defines.
  */
-#include "nvim/version_defs.h"
```

Comment:
(the position of this comment is irrelevant)

There's a reference to `version_defs.h`in `clint-ignored-files.txt` which should be removed.